atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/UpdateEvent.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.sweep.metrics;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public abstract class UpdateEvent {
+    public abstract String getTag();
+    public abstract String getLabel();
+
+    public static final UpdateEvent ONE_ITERATION = ImmutableUpdateEvent.builder()
```

Comment:
I still disagree with using tags for iteration vs full table metrics.  By default this is going to average both values on each graph.  I expect the iteration metrics to be insignificant compared to the full table metrics.  I can't think of a use case for combining the two metrics types at render time, or wanting to plot them against each other.  I think this is going to catch us and other users out.

Having them as separate metrics avoids all of this confusion.

Reply:
That makes perfect sense, I misunderstood how tagged metrics worked. Going to change this.